README.md

```diff
@@ -29,33 +29,37 @@ a development environment.
 
 Options:
 
-* `-h`, `--help`\
-  Print help information
-
-* `-i`, `--input-file <INPUT_FILE>`\
+* `-f`, `--input-file <INPUT_FILE>`\
   Path to an input file. If omitted, or equal to "-", read from standard
   input
 
-* `-l`, `--language <LANGUAGE>`\
-  Which language to parse and format [possible values: json, toml]
-
 * `-o`, `--output-file <OUTPUT_FILE>`\
   Path to an output file. If omitted, or equal to "-", write to standard
   output
 
+* `-i`, `--in-place`\
+  Format the input file in place. (This has the effect of setting the
+  output file equal to the input file.)
+
 * `-q`, `--query <QUERY>`\
   Which query file to use
 
+* `-l`, `--language <LANGUAGE>`\
+  Which language to parse and format [possible values: json, toml]
+
 * `-s`, `--skip-idempotence`\
   Do not check that formatting twice gives the same output
 
+* `-h`, `--help`\
+  Print help information
+
 * `-V`, `--version`\
   Print version information
 
 Language selection is based on precedence, in the following order:
+* A specified query file
 * A specified language
 * Detected from the input file's extension
-* A specified query file
```

Comment:
I don't think changing the order in which arguments are parsed changes the priority on language selection. Besides, I think it's better if we keep the current order (for performance reasons for ocaml files)

Reply:
It doesn't. I was just changing the order in the documentation (the README and the `--help`) to match. The priority order hasn't changed in this PR.

Reply:
I agree that the PR doesn't change the priority order, yet it changes the *documentation* about the priority order (the list just above).

Reply:
Because the documentation was wrong. The documentation had it as `language > extension > query`, whereas it should be `query > language > extension`. Right?

Reply:
If I'm not mistaken, `query > language > extension` used to be the priority order, but I changed it in #169 when I refactored the way languages are processed. Now query files have the lowest priority.
The reason for it is that `ocaml.scm` now covers two languages (ocaml interface and ocaml implementation), so its `#language!` predicate is interpreted as "try implementation first, then interface if it fails". Using that logic with the highest priority would be a performance hit, because most of the time we can actually know if we're processing implementation or interface by looking at the file extension, hence the priority change.

We can discuss whether this choice is correct of course, but I think it should be done in a separate issue

Reply:
Ah, my mistake. I remember you changed it, but I noticed [this comment](https://github.com/tweag/topiary/blob/e21c293acd905d11eb5472400b312a26047823a1/src/main.rs#L75) was inconsistent. I thought that was the correct order and the documentation was out-of-date... I'll fix it in the morning :+1:

Reply:
Changed in 33622f0bedc4